Design.md

```diff
@@ -0,0 +1,26 @@
+Overview
+    - This app gives players and teams an environment where they can seamlessly review and breakdown film from games / practices all in one place. It can be used for both internal development and for detailed scouting on future opponents.
+
+Features:
+1. Team Specific Accounts
+    - User notes with this team affiliation are all put in one place.
+    - This means scouting can be done without worrying about another team user seeing it.
+2. User Roles
+    - Users who are verified with a team and can post to that team's account
+    - Users who are not verified with a team and will only be posting on their personal account
+3. Watch Games in App
+    - In a perfect world, the video is playing in the same browser as the app is running.
+    - However, due to cose we might have to use external browser like youtube to run videos.
+        - If this is the case then make it easy to link film clip with timestamp of video
+4. Create Film Clips
+    - While watching a video, user is able to clip part of a video and make a note about what happens
+    - If this user is verified with a team, when another team member is watching the same video they will be able to see the note.
+    - Film clips are kept in a team-wide / user-wide database that can be searched / sorted by keywords (ie. players involved, topic discussed (ie. deep defense), teams, etc...)
+5. Allow Highlights to be Shared
+    - Plays that are deemed highlights while watching can easily be shared
+    - User has the option to include notes along with clip or if they only want to share the clip
+
+Rough Design
+![Home/Game Review](design-draft-images/home:game-review.jpg)
+![Search Home / Team Profile (other team)](design-draft-images/search-home:team-profile.jpg)
+![User Profile / Team Home (user's team)](design-draft-images/user-profile:team-home.jpg)
```

Comment:
User Profile:

1. A lot of this functionality is covered by the home page...do we actually need this page, or can we just add its functionality to home? Or more concretely, should we actually just have this be the home page?
2. I can think of a scenario where the user is tied to some sort of user settings, like if they want to use light mode or dark mode, for example. But that doesn't seem like what you've designed here.

Team Home (user's team):
1. What is the "scouting tab"? Is it a link? Why is it taking up so much space?
2. This is another page that i'm not convinced we need to have, especially not right away. I think if we just add some of the stuff on this page to the generic "team view" page, and lump in the announcements bit to the home page (or have some sort of inbox system), we can use just one page to display the user's team similarly to every other team.

Reply:
User Profile
1. I totally agree, opted to combine this with the home page.
2. I'm happy adding this into the navbar somewhere if you think its necessary.

Team Home
Cut this page. Added announcements to inbox and created an inbox page.